superset/models/helpers.py

```diff
@@ -875,7 +875,9 @@ def mutate_query_from_config(self, sql: str) -> str:
         """Apply config's SQL_QUERY_MUTATOR
 
         Typically adds comments to the query with context"""
-        if sql_query_mutator := config["SQL_QUERY_MUTATOR"]:
+        if (sql_query_mutator := config.get("SQL_QUERY_MUTATOR")) and not config.get(
```

Comment:
```suggestion
        if (sql_query_mutator := config["SQL_QUERY_MUTATOR"]) and not config["MUTATE_AFTER_SPLIT"]):
```

Reply:
@eschutho both these keys are defined in `config.py` and thus there's no need to use `.get()`.

Reply:
Thank you!